accounts/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.template.loader import get_template
+
+from house.constants import USER_LOGIN_PAGE_ROUTE, USER_SIGNUP_ROUTE
+
+
+class TestLoginPage:
+    def test_user_login_template(self):
+        get_template(USER_LOGIN_PAGE_ROUTE)
+
+    def test_login(self, client):
+        response = client.get('/accounts/login/')
+        assert response.status_code == 200
+        assert 'registration/login.html' in response.template_name
+
+
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template(USER_SIGNUP_ROUTE)
```

Comment:
Doesn't this function miss assert?

Reply:
Same : Nope, If the get template will fail the test will fail with a message of the Exception